src/redux/theme/theme.actions.js

```diff
@@ -0,0 +1,8 @@
+import { SET_THEME_MODE } from './theme.types';
+
+const setThemeMode = (boolean) => ({
```

Comment:
Please, rename `boolean` to something more meaningful with `is` prefix like `isDark`

Reply:
Done